crates/forge/src/scarb/config.rs

```diff
@@ -251,45 +174,45 @@ mod tests {
         }
     }
 
-    #[test]
-    fn test_fork_target_new_invalid_url() {
-        let name = "TestFork";
-        let url = "invalid_url";
-        let block_id_type = "number";
-        let block_id_value = "123";
-
-        let result = ForkTarget::new(name, url, block_id_type, block_id_value);
-        assert!(result.is_err());
-        assert_eq!(result.unwrap_err().to_string(), "Failed to parse fork url");
-    }
-
-    #[test]
-    fn test_fork_target_new_invalid_block_id_value_number() {
-        let name = "TestFork";
-        let url = "http://example.com";
-        let block_id_type = "number";
-        let block_id_value = "invalid_number";
-
-        let result = ForkTarget::new(name, url, block_id_type, block_id_value);
-        assert!(result.is_err());
-        assert_eq!(
-            result.unwrap_err().to_string(),
-            "Failed to parse block number"
-        );
-    }
-
-    #[test]
-    fn test_fork_target_new_invalid_block_id_value_hash() {
-        let name = "TestFork";
-        let url = "http://example.com";
-        let block_id_type = "hash";
-        let block_id_value = "invalid_hash";
-
-        let result = ForkTarget::new(name, url, block_id_type, block_id_value);
+    // Using rstest for parameterized invalid cases
+    #[rstest]
```

Comment:
We already have `test-case` dependency which we use with `#[test_case ...]` macro in other crates. I'd not add another dependency just for these tests.

Reply:
sorry, i miss that crate